drizzlepac/haputils/astrometric_utils.py

```diff
@@ -1813,6 +1818,30 @@ def build_wcscat(image, group_id, source_catalog):
     return wcs_catalogs
 
 
+def get_epoch(hdulist):
+    """Interpret observation date as epoch"""
+    d = hdulist[0].header['date-obs']
+    dt = datetime.strptime(d, '%Y-%m-%d')
+
+    return decimal_year(dt)
+
+
+def decimal_year(date):
+    def since_epoch(date): # returns seconds since epoch
+        return time.mktime(date.timetuple())
+    se = since_epoch
```

Comment:
Just a comment - No sure why you encapsulated the computation for seconds since the epoch inside of another function as SSE can be generally very useful.

Reply:
This is what happens when I blindly cut-and-paste from StackOverflow. ;-)  Actually, using 'astropy.time' to get the decimalyear is cleaner, and the change to use astropy's computation is included in 8f33c83.